dig/README.md

```diff
@@ -0,0 +1,78 @@
+# Dependency Injection Graph
+
+This package provides a fairly opinionated way of resolving object dependencies.
+There are two sides of dig: `Inject` and `Resolve`
```

Comment:
it's the same way to use it, one gives, one takes. If you have better ideas of how to phrase it let me know